src/jit/bitsetasshortlong.h

```diff
@@ -664,9 +664,11 @@ void BitSetOps</*BitSetType*/ BitSetShortLongRep,
                /*BitSetTraits*/ BitSetTraits>::ClearDLong(Env env, BitSetShortLongRep& bs)
 {
     assert(!IsShort(env));
-    // Recall that ClearD does *not* require "bs" to be of the current epoch.
-    // Therefore, we must allocate a new representation.
-    bs = MakeEmptyArrayBits(env);
```

Comment:
Ah, of course. That said, given that most of the methods do not work across Epochs, another option would be to run SPMI with a JIT that always uses the class implementation of BitSet.